tests/testsuite/workspaces.rs

```diff
@@ -2800,6 +2800,9 @@ Available binaries:
 [ERROR] no bin target named `crate2`
 
 [HELP] a target with a similar name exists: `crate1`
+[HELP] Available bin in `crate2` package:
+    crate2
+
```

Comment:
extra blank lines are back?

Reply:
Should be good to merge with this resolved.

Reply:
Also fixed. Thanks.